src/Microsoft.VisualStudio.LanguageServices.Razor/ViewImportChangeTracker.cs

```diff
@@ -0,0 +1,289 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using Microsoft.CodeAnalysis.Razor;
+using Microsoft.VisualStudio.Editor;
+using Microsoft.VisualStudio.Editor.Razor;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.TextManager.Interop;
+
+namespace Microsoft.VisualStudio.LanguageServices.Razor
+{
+    internal class ViewImportChangeTracker : IVsFileChangeEvents, IVsRunningDocTableEvents, IDisposable
```

Comment:
This was originally a private class in `DefaultVisualStudioOpenDocumentManager` but I moved this out and made it internal for the sake of testing and clarity.